### PR TITLE
Veja Ravine / Vieta Gorge / Cobalt Forest / Septyni Glen

### PR DESCRIPTION
**Pull request by terrapin47**

### CODE REVIEW

File: QUEST.tsv [hunk 1]
```diff
@@ -987,20 +987,20 @@ QUEST_20150317_000986	Reading Desk
...
-QUEST_20150317_001001	{nl}(There is something written, but it's all jumbled up and erased to be legible.)
-QUEST_20150317_001002	Unknown Date {nl}- {nl}Several days must've passed. {nl}I've been on the run for around two days now without food or sleep. {nl}The people who fled Klaipeda were all caught and killed. {nl}I don't even know how many are still alive and can walk. {nl}Everything is as quiet as the the shadows, yet moving at a high pace. We are all tense from fear of attack.
-QUEST_20150317_001003	They did not wish for any uproar or disorder. {nl}They are not people. {nl}We must somehow notify Klaipeda. {nl}A last resort would be to abandon everything we can't use and to move in.
+QUEST_20150317_001001	{nl}(There is something written, but it's all jumbled up and has ceased to be legible.)
+QUEST_20150317_001002	Unknown Date {nl}- {nl}Several days must've passed. {nl}I've been on the run for around two days now without food or sleep. {nl}The people who fled to Klaipeda were all caught and killed. {nl}I don't even know how many are still alive and can walk. {nl}Everything is as quiet as the the shadows, yet moving at a quick pace. We are all tense from fear of attack.
```
Comment (vtange) [inline QUEST.tsv]: quiet as the shadows

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -6277,20 +6277,20 @@ QUEST_LV_0100_20150317_006272	{memo X}
 QUEST_LV_0100_20150317_006273	Collect the venom of Black Maize.
 QUEST_LV_0100_20150317_006274	$Collect the venom of Black Maize as a gift for the Village Priest. It is only able to obtain the venom of Black Maize by using the paralysis powder while Black Maize's HP is down by half.
```
Comment (vtange) [inline QUEST_LV_0100.tsv]: Venom of Black Maize or Black Maize's Venom? Check the item name and match it

**Comment by zhouyu47** [comment]
Wouldn't just seeing "X 4" be a little confusing?

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**